tests/selenium/test_sizing.py

```diff
@@ -248,3 +250,78 @@ def test_szng002_percentages_result_in_same_widths(test):
     for i in range(1, len(variations)):
         table = test.driver.find_element_by_css_selector("#table{}".format(i))
         cells_are_same_width(target, table)
+
+
+@pytest.mark.parametrize(
+    "fixed_columns",
+    [
+        # dict(),
+        dict(fixed_columns=dict(headers=True)),
+        dict(fixed_columns=dict(headers=True, data=1)),
+    ],
+)
+@pytest.mark.parametrize(
+    "fixed_rows",
+    [
+        # dict(),
+        dict(fixed_rows=dict(headers=True)),
+        dict(fixed_rows=dict(headers=True, data=1)),
+    ],
+)
+@pytest.mark.parametrize(
+    "merge_duplicate_headers",
+    [dict(merge_duplicate_headers=True), dict(merge_duplicate_headers=False)],
+)
+@pytest.mark.parametrize(
+    "callback_props",
+    [
+        dict(
+            data=[
+                {"_": 0, "a": 85, "b": 601, "c": 891},
+                {"_": 0, "a": 967, "b": 189, "c": 514},
+                {"_": 0, "a": 398, "b": 262, "c": 743},
+                {
+                    "_": "SOME VERY LONG VALUE",
+                    "a": "SOME VERY LONG VALUE 2",
+                    "b": "SOME VERY LONG VALUE 3",
+                    "c": "SOME VERY LONG VALUE 4",
+                },
+                {"_": 0, "a": 89, "b": 560, "c": 582},
+                {"_": 0, "a": 809, "b": 591, "c": 511},
+            ]
+        ),
+        dict(merge_duplicate_headers=True),
+        # dict(fixed_rows=dict(headers=True)),
```

Comment:
Commenting out certain, more redundant, scenarios. With the above, this reduces the number of tests combinations from 126 to 40, which is more palatable in terms of CI time.